cms/envs/common.py

```diff
@@ -41,7 +41,7 @@
     # indirectly accessed through the email opt-in API, which is
     # technically accessible through the CMS via legacy URLs.
     PROFILE_IMAGE_BACKEND, PROFILE_IMAGE_DEFAULT_FILENAME, PROFILE_IMAGE_DEFAULT_FILE_EXTENSION,
-    PROFILE_IMAGE_SECRET_KEY,
+    PROFILE_IMAGE_SECRET_KEY, PROFILE_IMAGE_MIN_BYTES, PROFILE_IMAGE_MAX_BYTES,
```

Comment:
@muhammad-ammar @cahrens - note that this ugly import hack is needed to allow profile_images to use PROFILE_IMAGE_MAX_BYTES. The problem is that profile_images is in openedx and so needs to work in both LMS and CMS, and hence the setting needs to be available in CMS too.